databricks/koalas/indexes.py

```diff
@@ -1358,6 +1457,12 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
 
     value_counts.__doc__ = IndexOpsMixin.value_counts.__doc__
 
+    def argmax(self):
+        raise TypeError("reduction operation 'argmax' not allowed for this dtype")
```

Comment:
Just for clarification, it's matched to pandas' multi-index right?

Reply:
yes you're right. :)

```python
>>> pmidx = pd.MultiIndex.from_tuples([('a', 'x'), ('b', 'y')])
>>> pmidx.argmax()
Traceback (most recent call last):
...
TypeError: reduction operation 'argmax' not allowed for this dtype
```